docs/manage-metadata-using-gravitino.md

```diff
@@ -733,6 +733,159 @@ In addition to the basic settings, Gravitino supports the following features:
+| `day`             | Extract a date or timestamp day, as days from 1970-01-01     | date, timestamp, timestamptz                                                        | int         | `{"strategy":"day","fieldName":["score"]}`                       | `Transforms.day("score")`                       | `PARTITION BY day(score)`          |
+| `month`           | Extract a date or timestamp month, as months from 1970-01-01 | date, timestamp, timestamptz                                                        | int         | `{"strategy":"month","fieldName":["score"]}`                     | `Transforms.month("score")`                     | `PARTITION BY month(score)`        |
+| `year`            | Extract a date or timestamp year, as years from 1970         | date, timestamp, timestamptz                                                        | int         | `{"strategy":"year","fieldName":["score"]}`                      | `Transforms.year("score")`                      | `PARTITION BY year(score)`         |
+| `bucket[N]`       | Hash of value, mod N                                         | int, long, decimal, date, time, timestamp, timestamptz, string, uuid, fixed, binary | int         | `{"strategy":"bucket","numBuckets":10,"fieldNames":[["score"]]}` | `Transforms.bucket(10, "score")`                | `PARTITION BY bucket(10, score)`   |
+| `truncate[W]`     | Value truncated to width W                                   | int, long, decimal, string                                                          | Source type | `{"strategy":"truncate","width":20,"fieldName":["score"]}`       | `Transforms.truncate(20, "score")`              | `PARTITION BY truncate(20, score)` |
+| `list`            | Partition the table by a list value                          | Any                                                                                 | Any         | `{"strategy":"list","fieldNames":[["dt"],["city"]]}`             | `Transforms.list(new String[] {"dt", "city"})`  | `PARTITION BY list(dt, city)`      |
+| `range`           | Partition the table by a range value                         | Any                                                                                 | Any         | `{"strategy":"range","fieldName":["dt"]}`                        | `Transforms.range(20, "score")`                 | `PARTITION BY range(score)`        |
+
+Except the strategies above, you can use other functions strategies to partition the table, for example, the strategy can be `{"strategy":"functionName","fieldName":["score"]}`. The `functionName` can be any function name that you can use in SQL, for example, `{"strategy":"toDate","fieldName":["score"]}` is equivalent to `PARTITION BY toDate(score)` in SQL.
```

Comment:
You should add this to the document.
```
All transforms must return null for a null input value.
```

Reply:
I'm not so sure about this point as it's only appliable to Iceberg currently, I need to check this for `Hive`.

Reply:
OK, if we don't follow this, should we explain null input?

Reply:
Got it.

Reply:
> I'm not so sure about this point as it's only appliable to Iceberg currently, I need to check this for `Hive`.

Can you help confirm this, @mchades?

Reply:
I believe this is not determined by Gravitino, but rather depends on the underlying catalog.